site/utils/siri.ts

```diff
@@ -8,11 +8,56 @@ import {
     Period,
 } from "@create-disruptions-data/shared-ts/siriTypes";
 import dayjs from "dayjs";
-import { getDatetimeFromDateAndTime } from "./dates";
+import { getDatetimeFromDateAndTime, getFormattedDate } from "./dates";
 import { Validity } from "../schemas/create-disruption.schema";
 import { Disruption } from "../schemas/disruption.schema";
 
-const getValidityPeriod = (period: Validity): Period => ({
+const getValidityPeriod = (period: Validity): Period[] => {
```

Comment:
Do we have any unit tests which cover this functionality? There is quite a lot of business logic in here, slightly concerned we might not have tests for it.

Reply:
done